Update installing-stencil.md (#1259)
bigcommerce/dev-docs: docs/stencil-docs/installing-stencil-cli/installing-stencil.md

## Installing on Mac

> To install Stencil CLI and it's dependencies on Mac, open a terminal and run the following commands. Refer to [Stencil CLI README.MD](https://github.com/bigcommerce/stencil-cli) for latest `node` version supported.

Reviewer: ```suggestion
To install Stencil CLI and its dependencies on Mac, open a terminal and run the following commands. Refer to [Stencil CLI README.MD](https://github.com/bigcommerce/stencil-cli) for latest `node` version supported.

```
Changed contraction to possessive pronoun.

# Install Stencil CLI
npm install -g @bigcommerce/stencil-cli
```
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme: info -->

> Use these instructions for Mac's with a M1 chip.

</div>
</div>
</div>

```shell

# Install Node Version Manager (nvm)
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash

# Install Stencil CLI supported version of Node.js
nvm install 12.1

> # Switch to Stencil CLI supported version of Node.js:

Reviewer: ```suggestion
# Switch to Stencil CLI supported version of Node.js

```

nvm use 12.1

# Install Stencil CLI
npm install -g @bigcommerce/stencil-cli
```

## Installing on Windows

> There's two methods for installing Stencil CLI and its dependencies on Windows.

Reviewer: ```suggestion
There are two methods for installing Stencil CLI and its dependencies on Windows.

```
